tox.ini

```diff
@@ -43,15 +43,15 @@ extras = dev
 allowlist_externals = mkdir
 commands_pre = mkdir -p {tox_root}/results
 
-[testenv:{unit,integration}-py3.{8,9,10,11,12}]  # Configuration for all tests using pytest
+[testenv:{unit,integration}-py3.{10,11,12}]  # Configuration for all tests using pytest
 base = testenv, test
 description =
     unit: Run unit tests with pytest
     integration: Run integration tests with pytest
 labels =
-    py3.{8,10,11}: tests
-    unit-py3.{8,10,11}: unit-tests
-    integration-py3.{8,10,11}: integration-tests
+    py3.{10,12}: tests
+    unit-py3.{10,12}: unit-tests
+    integration-py3.{10,12}: integration-tests
```

Comment:
Should we run here also against `3.11` ?

Reply:
Here we should include all the versions.

Reply:
Changed to include also `3.11`